src/pkg/packaging/dir.props

```diff
@@ -9,26 +9,27 @@
     <SharedHostPublishRoot>$(IntermediateOutputRootPath)sharedHost/</SharedHostPublishRoot>
     <HostFxrPublishRoot>$(IntermediateOutputRootPath)hostFxr/</HostFxrPublishRoot>
     <SharedFrameworkPublishRoot>$(IntermediateOutputRootPath)sharedFx/</SharedFrameworkPublishRoot>
-    <CombinedPublishRoot>$(IntermediateOutputRootPath)combined-framework-host/</CombinedPublishRoot>   
+    <CombinedPublishRoot>$(IntermediateOutputRootPath)combined-framework-host/</CombinedPublishRoot>
   </PropertyGroup>
 
   <PropertyGroup>
     <ProductMoniker>$(PackageTargetRid)-$(HostFullVersion)</ProductMoniker>
+    <PortableMoniker Condition="'$(PortableBuild)' == 'true'">-portable</PortableMoniker>
```

Comment:
All of our builds are now "portable". I don't think we need this in our built assets anymore.

@gkhanna79 - thoughts?